_posts/2017-03-16-two-months-of-tech-interviews.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: "Two Months Of Tech Interviews 💼"
+comments: true
+categories: tech recurse job
+---
+
+The time finally came. 
+
+Full. Time. Job. Search.
+
+Over the last 5 years, I've accumulated over 2.5 years of paid software engineering experience. I've also accumulated a fair amount of self-directed experience.
+
+But, I've never accepted a full-time job...
+
+> Sorry, I have more school to finish.
+
+> Sorry, I want to work in a different city.
+
+> Sorry, I want to travel and focus on my own projects.
+
+> Sorry, I want to attend the Recurse Center.
+
+...until now.
+
+On Monday, I accepted a full-time offer with [Squarespace](https://www.squarespace.com/) 🎉.
+
+Job hunting was far from the seamless process I imagined it would be. Here's a review of my 2 months interviewing in tech.
+
+### Interview prep
+
+The first step of my journey was interview was prep.
+
+In December I was finishing a batch at the Recurse Center. During the final weeks I switched gears towards interview prep.
+
+I highly recommend taking a week or two to prep. Most of the time between mid-December to early-January was just switching my brain to interview mode.
+
+During this time, I made the mistake of scheduling a phone interview before I was fully confident. I failed pretty spectacularly and ruined a really great opportunity... not good 😢.
+
+### Resume cleanup
+
+I updated [my resume](/resume) before starting applications and a few times throughout the process. Some things I tried to do:
+
+- **One page**: Companies get tons of resumes and they have limited time to read each one.
+- **Use bullets**: They are easier to skim and help separate discreet accomplishments.
+- **Show, don't tell**: Let experiences speak for themselves. Building something with a 10/10 rating is better than saying "my project was popular".
+- **Less is more**: At the start I thought more things meant more impressive. Later on I realized that each item has diminishing returns. Too much just creates clutter.
+- **Consistency:** I tried to be consistent with how I mention technologies and structure items.
+
+With each update I shared my resume with a bunch of people for feedback. Here's a .gif of all the iterations it went through.
+
+![Resumes 2017](/assets/posts/resumes-2017.gif){: .center-image }
```

Comment:
Nice visualization. Maybe add dates or version numbers to the gif, if that is easily possible? Otherwise it's hard to see, seeing it the first time, when it started or when it loops.